### ISSUE TITLE
Text missing on top level slides if 2d presentations are enabled

### ISSUE THREAD

**Issue by stefanfritsch** [rstudio/revealjs#40]
Hi,

based on the official [example](http://lab.hakim.se/reveal-js/?autoSlide=5000&loop=true&controls=false#/2) I'd expect to be able to output text on both top level slides as well as basement slides. But in rmarkdown only basement level slides have text, e.g. in the following the only text that is actually in the result is the headers and "even more text".

```

---
title: "something"
author: "me"
date: "19 September 2016"
output: revealjs::revealjs_presentation

---

# Slide 1

Test
* test
* test
* test

# Slide 2

more text

## subslide 1

even more text
```

I looked in the source  and the text is simply not in the html, it is not just hidden.

R: 3.3.1
revealjs: current github version

Thx
Stefan

**Comment by ekstroem** [comment]
I'm experiencing this as way. Not sure if it's a `pandoc` issue.

**Comment by vspinu** [comment]
Any workaround for this?

**Comment by vspinu** [comment]
This is pandoc issue and it's by design for compatibility with other presentation styles. See this SO [answer](http://stackoverflow.com/a/31778080/453735).

**Comment by ekstroem** [comment]
Nice find @vspinu ! Guess we can close this for now as it's not a revealjs issue.

**Comment by vspinu** [comment]
There is an open [issue](https://github.com/jgm/pandoc/issues/2265) for pandoc exactly for this problem.

**Comment by vspinu** [comment]
The workaround is to squeeze some html yourself:

```md
---
title    : Title
date     : Dec 22, 2016
output   : 
    revealjs::revealjs_presentation
---

<section class="titleslide level1">
<h1>TOP SLIDE</h1>

Some blabla

## SUBSLIDE

Some sub-blabla
</section>
```

**Comment by quiestrho** [comment]
The workaround unfortunately works only for the first horizontal slide. Sad that it happens so. It defies the purpose of using 2d presentations. I want to have horizontal slides as my main presentation and vertical ones as extra material to show depending on the interest in the audience.

**Comment by vspinu** [comment]
> The workaround unfortunately works only for the first horizontal slide. 

I don't remember the details anymore, since it has been a while when I used revealjs, but I am pretty confident that it worked fine for all slides.

 In any case, you should be able to tweak all possible issues with html. You can write the whole presentation in html and avoid pandoc completely.

**Comment by jthurner** [comment]
Fixed in [pandoc 2.7](https://pandoc.org/releases.html#pandoc-2.7-3-march-2019)

**Comment by cderv** [comment]
I confirm this is now fixed since Pandoc 2.7
